telemetry/definitions/commonDefinitions.json

```diff
@@ -1659,6 +1659,11 @@
             "name": "xrayEnabled",
             "type": "boolean",
             "description": "Whether or not AWS X-Ray is enabled"
+        },
+        {
+            "name": "appTemplateSelected",
```

Comment:
Is there a reason you made this a metric on it's own? Can we just include this in the metadata of "app_walkthroughSelectTemplate"?

Reply:
This is a metadata. We need to define metadata in order to use them in metrics.

Reply:
I mean, can we not say inside of the "app_walkthroughSelectTemplate" metric 

```
"metadata": [
                {
                    "type": "template",
                    "required": true
                },
```
and have the value of "template": "s3_resizing" / "rest_api_hello_world" or something like that?